.github/workflows/ci.yml

```diff
@@ -47,7 +47,9 @@ jobs:
       - run: yarn install --frozen-lockfile
 
       - run: yarn shadow-cljs release npm
+      - run: yarn run bundle-ts
```

Comment:
Please add this to the end of the `build` and `build:dev` scripts in the package.json for consistency. 
Please also check that the CD workflow is also building types.
And you can omit the `run` with yarn `yarn bundle-ts`.

Reply:
@smothers I think I resolved everything besides CD. We should talk re the CD workflow because it's unclear to me what it's doing.